CHANGELOG_PENDING.md

```diff
@@ -4,3 +4,10 @@
  [#776](https://github.com/pulumi/pulumi-java/pull/776)
 
 ### Bug Fixes
+
+- [sdk] [#774](https://github.com/pulumi/pulumi-java/issues/774) 
+  fixes deserialization regression in new codegen specific part of SDK, 
+  complex @CustomType's are deserializable now
```

Comment:
Seems duplicative changelog but I can tidy it up later before releasing

Reply:
yes, there were two regressions, the second one was just a debug message

Reply:
sorry, I missed it

Reply:
should be ok now